Fang-4-Group/Graduation-Project
# [GP-122] Add column and edit insert data

## Changes
- Add two column(Line_ID, Mail) in PEOPLE table
- Add sample data

## Test
1. run `docker-compose up -d`
2. click http://localhost:7877/pg_init/
3. click http://localhost:7877/get_young_info/?amount=3
if success you will see Line_ID and Mail in response
<img width="1392" alt="截圖 2024-07-27 下午1 46 24" src="https://github.com/user-attachments/assets/7e76e8db-9557-428e-a64d-9272423ec60d">
